src/Browsershot.php

```diff
@@ -113,6 +114,17 @@ public function setChromePath(string $executablePath)
         return $this;
     }
 
+    public function post(array $postParams = [])
+    {
+        if (array_keys($postParams) === range(0, count($postParams) - 1)) {
```

Comment:
What is being checked here? Put the result in a descriptive variable like `$arrayIsFullyKeyed`.

Reply:
As I understand now, we can safely remove this check at all, since there is nothing technically wrong with such query as '0=foo&1=bar'. But thinking for the user is often superfluous. So it was my overthinking.